rest-api-spec/src/main/resources/rest-api-spec/api/indices.analyze.json

```diff
@@ -15,16 +15,6 @@
         "index": {
```

Comment:
I've left the "index" parameter in here, despite it being a path param, as IIUC then path params and URL params get collapsed into a single params object in the rest handler.

Reply:
this is correct, index should be left as-is.